data/json/recipes/practice/tailoring.json

```diff
@@ -0,0 +1,18 @@
+[
+  {
+    "id": "prac_leather_beg",
+    "type": "practice",
+    "activity_level": "NO_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_TAILORING",
+    "name": "leatherworking (beginner)",
+    "description": "Practice basic principles of tailoring with leathers, furs, and hides.",
+    "skill_used": "tailor",
+    "proficiencies": [ { "proficiency": "prof_leatherworking_basic", "fail_multiplier": 1, "time_multiplier": 1 } ],
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 1, "max_difficulty": 2, "skill_limit": 4 },
```

Comment:
With `max_difficulty` at 2 the effective skill limit is 3. Setting it to 4 has no effect, but may cause some confusion in the UI. "This practice action will not increase your tailoring skill above 4" is technically correct but can be taken to imply that practicing can raise the skill to 4, which is false.
I recommend setting it to 3 or removing it.

Reply:
Thanks for clarifying. I think I'll increase `max_difficulty` to 3 here, since it's auto-learned at tailoring level 3. I included a note in the docs about this relationship as well.

Reply:
I went with your suggestion of skill_limt 3 here.